inc/front/plugin-compatibility.php

```diff
@@ -158,25 +158,6 @@ function __rocket_cdn_on_sliders_with_lazyload() {
 }
 
 /**
- * Conflict with WP Retina x2: Apply CDN on srcset attribute.
- *
- * @since 2.5.5
- */
-add_filter( 'wr2x_img_retina_url', '__rocket_cdn_on_images_from_wp_retina_x2' );
-add_filter( 'wr2x_img_url', '__rocket_cdn_on_images_from_wp_retina_x2' );
-function __rocket_cdn_on_images_from_wp_retina_x2( $url ) {
```

Comment:
Don't forget to deprecated this function __rocket_cdn_on_images_from_wp_retina_x2() as it's renamed rocket_cdn_on_images_from_wp_retina_x2() without __ at the beginning